### PR TITLE
add optimize

### PR DESCRIPTION
**Pull request by zmaslova**
Same MR as https://github.com/openvinotoolkit/mmdetection/pull/159
Performance on CIFAR dataset

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/zmaslova/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/zmaslova/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

model name | original | IR | Optimized IR
-- | -- | -- | --
mobilenet_v3_small | 0.8321 | 0.8322 | 0.8249
mobilenet_v3_large_075 | 0.8518 | 0.8519 | 0.8481
mobilenet_v3_large_1 | 0.8568 | 0.8569 | 0.8529

</body>

</html>

FPS by benchmark app

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/zmaslova/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/zmaslova/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

model name | IR | Optimized IR
-- | -- | --
mobilenet_v3_small | 3563.52 | 4931.99
mobilenet_v3_large_075 | 1962.6 | 2899.78
mobilenet_v3_large_1 | 1591.4 | 2361.95

</body>

</html>

### CODE REVIEW

**Comment by zmaslova** [comment]
cc @MaximProshin

**Comment by MaximProshin** [comment]
@zmaslova , do you have measurements for mobilenetv3 models?

**Comment by zmaslova** [comment]
> do you have measurements for mobilenetv3 models?

I have only efficientnet_b0 pretrained model

**Comment by MaximProshin** [comment]
@kshpv , can you help Zoya to find mobilenetv3?

**Comment by kshpv** [comment]
@zmaslova contact me pls

**Comment by zmaslova** [comment]
PR is updated, also added measurements for mobilenetv3

**Comment by MaximProshin** [comment]
@zmaslova , I see significant accuracy drop for these models (>1%) and I assume it's due to DQ applied while these (at least some of them) models should have custom configs. @kshpv, can you share POT configs with Zoya which we used to measure accuracy and performance before. Finally, we should achieve <1% accuracy drop,

**Comment by zmaslova** [comment]
> @zmaslova , I see significant accuracy drop for these models (>1%) and I assume it's due to DQ applied while these (at least some of them) models should have custom configs. @kshpv, can you share POT configs with Zoya which we used to measure accuracy and performance before. Finally, we should achieve <1% accuracy drop,

@MaximProshin Updated (without efficientnet, waiting for @kshpv update )

**Comment by zmaslova** [comment]
I suggest PR can be merged without one model result

File: torchreid/integration/sc/openvino_task.py [hunk 1]
```diff
@@ -22,20 +26,35 @@
...
+)
+
 from sc_sdk.entities.annotation import AnnotationScene
 from sc_sdk.entities.datasets import Dataset
-from sc_sdk.entities.label import Label
 from sc_sdk.entities.media_identifier import ImageIdentifier
-from sc_sdk.entities.resultset import ResultSet
+from sc_sdk.logging import logger_factory
+
+from compression.api import DataLoader
```
Comment (sovrasov) [inline torchreid/integration/sc/openvino_task.py:49]: From which pypi package this import is available?

Comment (zmaslova) [reply torchreid/integration/sc/openvino_task.py:49]: openvino-dev==2021.4

**Review by sovrasov — APPROVED** [review]

**Pull request merged**